tests/frontend/matcher/test_image_matchers.py

```diff
@@ -0,0 +1,35 @@
+"""Unit tests for image matchers
+
+Authors: Ayush Baid
+"""
+from pathlib import Path
+import unittest
+
+import parameterized
+
+from gtsfm.loader.olsson_loader import OlssonLoader
+from gtsfm.frontend.matcher.loftr import LOFTR
+
+# defining the path for test data
+DATA_ROOT_PATH = Path(__file__).resolve().parent.parent.parent / "data"
+TEST_DATA_PATH = DATA_ROOT_PATH / "set1_lund_door"
+
+
+@parameterized.parameterized_class(
```

Comment:
What do you think about this pattern for testing?

Reply:
looks good.